index.html

```diff
@@ -8,7 +8,24 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
-  <h1 class="title">Hello World</h1> 
+<body class="body">
+  
+  <div class="title-container">
+	  <img class="triangle" src="assets/triangle.svg" alt="triangle">
+	  <h1 class="title">Front-End Tools	</h1> 
+  </div>
+
+
+  <div class="tech">
+
+	<img class="logo" src="assets/react_icon.svg" alt="react">
+	<p class="name">ReactJS</p>
+	<p class="release"> Release: 2013</p>
+	<p class="description">React is a declarative, component based JavaScript library for building user interfaces.</p>
```

Comment:
Try to use the strings in the same line with the tag or in the next, but don´t use both ways.